AppCenterIdentity/AppCenterIdentity/Internals/Ingestion/MSIdentityConfigIngestion.m

```diff
@@ -0,0 +1,61 @@
+#import "MSIdentityConfigIngestion.h"
+#import "MSAppCenter.h"
+#import "MSAppCenterInternal.h"
+#import "MSHttpIngestionPrivate.h"
+#import "MSIdentityPrivate.h"
+#import "MSLoggerInternal.h"
+
+@implementation MSIdentityConfigIngestion
+
+- (id)initWithBaseUrl:(NSString *)baseUrl appSecret:(NSString *)appSecret {
+  NSString *apiPath = [NSString stringWithFormat:@"/identity/%@.json", appSecret];
+  if ((self = [super initWithBaseUrl:baseUrl
+                             apiPath:apiPath
+                             headers:nil
+                        queryStrings:nil
+                        reachability:[MS_Reachability reachabilityForInternetConnection]
+                      retryIntervals:@[ @(10), @(5 * 60), @(20 * 60) ]])) {
```

Comment:
Let's put the retry intervals in a constants file

Reply:
We can add a default constant or make another overridden method that doesn't take `retryInterval` and use default. I'd prefer the second option.

Reply:
I'm fine with that